_Classes/DeusEx/Classes/HUDAmmoDisplay.uc

```diff
@@ -104,16 +105,20 @@ event DrawWindow(GC gc)
 			gc.SetTextColor(colAmmoText);
 
 		// Ammo count drawn differently depending on user's setting
-		if (weapon.ReloadCount > 1 || weapon.IsA('WeaponGEPGun'))
+		if (weapon.ReloadCount > 1 || weapon.IsA('WeaponGEPGun') || weapon.AmmoName == Class'Ammo20mm')
```

Comment:
Find a generic way to do this. Maybe a weapon property. Althought it may not be necessary at all.

Reply:
Possibly? There's only one ammo type and one weapon as an exception, unless we add more weapons in the future (which I doubt) there won't be a need to make this generic